django_socio_grpc/servicer_proxy.py

```diff
@@ -56,6 +58,7 @@ def handler(request, context):
                 db.reset_queries()
                 # INFO - AM - 22/04/2021 - next line break tests. Need to more understand the drowback about memory in production
                 # db.close_old_connections()
+                close_old_connections()
```

Comment:
remove the comments aboves.
First comment change it to:
`# INFO - AM - 30/06/2021 - Need this in production environnement to avoid SSL end of files errors when too much connection on database`